plugins/jira/jira.plugin.zsh

```diff
@@ -64,7 +64,7 @@ function jira() {
       local issue_arg=$(git rev-parse --abbrev-ref HEAD)
       local issue="${jira_prefix}${issue_arg}"
     else
-      local issue_arg=$action
+      local issue_arg=$( tr '[a-z]' '[A-Z]' <<< $action)
```

Comment:
You can uppercase a variable with this zsh syntax: `${(U)var}`, so you can get rid of the `tr` call and just use `${(U)action}`.